### DISCUSSION TITLE
Parse INSPIRE XML in geotools

### DISCUSSION THREAD

**Discussion by MarekSven** [INSPIRE-MIF/helpdesk#155] [Q&A]
Hi, 
I wanted to parse the DB Netze XML file, which is based on the Inspire model, in Java using GeoTools (I tried versions 22.2, 26.7, 28.3 with the same result). 

When running a simple example file, I am however getting XSD parsing errors:

WARNING: Error parsing: http://inspire.ec.europa.eu/schemas/tn-ra/3.0/RailwayTransportNetwork.xsd
WARNING: Error parsing: http://inspire.ec.europa.eu/schemas/base/3.2/BaseTypes.xsd
WARNING: Error parsing: http://inspire.ec.europa.eu/schemas/gn/3.0/GeographicalNames.xsd
WARNING: Error parsing: http://inspire.ec.europa.eu/schemas/net/3.2/Network.xsd
WARNING: Error parsing: http://inspire.ec.europa.eu/schemas/tn/3.0/CommonTransportElements.xsd
WARNING: Error parsing: http://inspire.ec.europa.eu/schemas/tn-ro/3.0/RoadTransportNetwork.xsd

Does anyone knows which pecularities are bound with parsing of those files?

**Comment by fabiovinci** [comment]
Dear @MarekSven,

it may be due to the different namespace used in these old schemas, please try using the new ones:

- http://inspire.ec.europa.eu/schemas/tn-ra/4.0/RailwayTransportNetwork.xsd
- http://inspire.ec.europa.eu/schemas/base/3.3/BaseTypes.xsd
- http://inspire.ec.europa.eu/schemas/gn/4.0/GeographicalNames.xsd
- http://inspire.ec.europa.eu/schemas/net/4.0/Network.xsd
- http://inspire.ec.europa.eu/schemas/tn/4.0/CommonTransportElements.xsd
- http://inspire.ec.europa.eu/schemas/tn-ro/4.0/RoadTransportNetwork.xsd
